src/treemodel/delta/delta.js

```diff
@@ -118,6 +131,31 @@ export default class Delta {
 	}
 
 	/**
+	 * Custom toJSON method to make deltas serializable.
+	 *
+	 * @returns {Object} Clone of this delta with added class name.
+	 */
+	toJSON() {
+		let json = clone( this );
+
+		json.__class = this.constructor.className;
+
+		json.batch = null;
+
+		return json;
+	}
+
+	/**
+	 * Delta class name. Used by {@link engine.treeModel.delta.Delta#toJSON} method for serialization and
+	 * {@link engine.treeModel.delta.Delta.fromJSON} during deserialization.
+	 *
+	 * @type {String}
```

Comment:
`@static` is not needed.
![selection_020](https://cloud.githubusercontent.com/assets/247363/15323651/66da0bda-1c43-11e6-852e-b1a77d2ead0a.png)